temp/legacy-code/src/main/java/com/ivy/legacy/utils/DateExt.kt

```diff
@@ -85,14 +85,10 @@ fun LocalDateTime.formatNicelyWithTime(
     val today = dateNowUTC()
     val isThisYear = today.year == this.year
 
-    val patternNoWeekDay = "dd MMM HH:mm"
+    val patternNoWeekDay = "yyyyMMdd_HH:mm"
```

Comment:
So not required for export and report right because noWeekday boolean true only for backup, export, reports.

Reply:
But don't change the default value for some globally used util. This will lead to unexpected implications throughout the entire codebase. Just pass it as a parameter.